src/redisraft.c

```diff
@@ -1716,7 +1774,7 @@ static void handleInfo(RedisModuleInfoCtx *ctx, int for_crash_report)
     RedisModule_InfoAddFieldULongLong(ctx, "snapshots_created", rr->snapshots_created);
 
     RedisModule_InfoAddSection(ctx, "clients");
-    RedisModule_InfoAddFieldULongLong(ctx, "clients_in_multi_state", MultiClientStateCount(rr));
+    RedisModule_InfoAddFieldULongLong(ctx, "clients_state_tracking", ClientStateCount(rr));
```

Comment:
Right now, this field becomes "connected client count". So, if I'm not mistaken, this will be same with `connected_clients` field in INFO. 

Previously, it was "clients in multi state" and a test uses it. Maybe, we loop over clients and count clients in multi state ?

If we can run/verify test in some other way, it might also mean we don't need this field. @yossigo Do you think this is useful? Redis doesn't report clients in multi state if I'm not mistaken.

Reply:
This comment is still valid.

Reply:
right, I didn't remove it, because felt it needed discussion.  outdated is because I had to rebase.

Reply:
removed as discussed in zoom meeting